pkg/boilerplate/controller/filter.go

```diff
@@ -67,3 +67,17 @@ func FilterByNames(parentFunc ParentFunc, names ...string) ParentFilter {
 		DeleteFunc: has,
 	}
 }
+
+func AcceptAll(parentFunc ParentFunc) ParentFilter {
+	has := func(obj v1.Object) bool {
+		return true
+	}
```

Comment:
This will be noisy... do we want to kick our sync loop for ANY config changed in the openshift config namespace?